classes/PrestashopConfiguration.php

```diff
@@ -78,7 +78,7 @@ public function getPrestaShopVersion()
             $this->psRootDir . '/app/AppKernel.php',
         );
         foreach ($files as $file) {
-            $version = $this->findPrestaShopVersionInFile(file_get_contents($file));
+            $version = $this->findPrestaShopVersionInFile(@file_get_contents($file));
```

Comment:
I'm more in favor of checking [the file exists](https://www.php.net/manual/en/function.file-exists.php), instead of hiding the warning.

Can you please update your PR to go on if the file is not present on the filesystem?

```suggestion
            if (!file_exists($file)) {
                continue;
            }
            $version = $this->findPrestaShopVersionInFile(file_get_contents($file));
```

Reply:
It's changed